### PR TITLE
Core/proper pybind namespace app template

### PR DESCRIPTION
**Pull request by philbucher**
Related to #2326 

I also updated the formatting (separate commit)

### CODE REVIEW

File: kratos/templates/template_application/custom_python/add_custom_strategies_to_python.cpp.in [hunk 1]
```diff
@@ -32,34 +32,30 @@
...
-// 				.def("MoveNodes",&TestStrategy< SparseSpaceType, LocalSpaceType, LinearSolverType >::MoveNodes)
-// 				;
+    typedef LinearSolver<SparseSpaceType, LocalSpaceType > LinearSolverType;
+    typedef SolvingStrategy< SparseSpaceType, LocalSpaceType, LinearSolverType > BaseSolvingStrategyType;
+    typedef Scheme< SparseSpaceType, LocalSpaceType > BaseSchemeType;
 
-		}
+    //********************************************************************
+    //********************************************************************
+    //  class_< TestStrategy< SparseSpaceType, LocalSpaceType, LinearSolverType>,
```
Comment (roigcarlo) [inline kratos/templates/template_application/custom_python/add_custom_strategies_to_python.cpp.in]: I think that should be `py::class_`

**Comment by philbucher** [comment]
@roigcarlo I tested it in #2491 , now this should also be ok

**Review by roigcarlo — DISMISSED** [review]
:+1:

**Comment by philbucher** [comment]
@roigcarlo I moved the namespace-declaration inside the functions based on @RiccardoRossi s suggestion in #2491

**Review by roigcarlo — APPROVED** [review]
Cool :+1:

**Pull request merged**